Correct bugs in scripts

jsf9k opened the pull request:
## 🗣 Description

This pull request fixes some bugs/typos in the `setup_freeipa_replica.sh` and `setup_freeipa_server.sh` scripts.

## 💭 Motivation and Context

The scripts do not work properly without these fixes.

## 🧪 Testing

I successfully installed a FreeIPA server with these changes.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

jsf9k committed:
Add missing "FILE" in pkinit_anchors hack

jsf9k committed:
Fix ipa certmaprule-add syntax

jsf9k committed:
Merge branch 'develop' into improvement/add-certs-and-scripts

dav3r reviewed (approved):
Hasta la vista buggies! 😎

jsf9k committed:
Force replicas to setup a CA

This is the key step that makes masters and replicas indistinguishable
once they are up and running.

jsf9k committed:
Merge branch 'improvement/add-certs-and-scripts' of github.com:cisagov/ansible-role-freeipa-server into improvement/add-certs-and-scripts

hillaryj reviewed (approved):
👣 💥
